tor_archivist/main.py

```diff
@@ -98,9 +100,16 @@ def run(cfg):
             continue
 
         # the original post that might have been transcribed
-        original_post = config.r.submission(url=post.url)
+        try:
+            logging.debug("Fetching original post from url")
+            original_post = config.r.submission(url=post.url)
+        except prawcore.exceptions.Forbidden:
+            logging.warning("Unable to retrieve the original submission. Skipping archiving of post")
+            original_post = None
+            continue
```

Comment:
Do we still want to have this continue statement here? It will stop processing the current element and won't use the new flag if I'm reading it right.

Reply:
oops! Nope I don't